interpreter/cling/include/cling/Interpreter/DynamicLibraryManager.h

```diff
@@ -74,21 +74,28 @@ namespace cling {
     ///\brief Concatenates current include paths and the system include paths
     /// and performs a lookup for the filename.
     ///\param[in] libStem - The filename being looked up
+    ///\param[in] RPATH - Library specific RPATH
+    ///\param[in] RUNPATH - Library specific RUNPATH
     ///
     ///\returns the canonical path to the file or empty string if not found
     ///
-    std::string lookupLibInPaths(llvm::StringRef libStem) const;
-
+    std::string lookupLibInPaths(llvm::StringRef libStem,
+                                 const std::string& RPATH = "",
+                                 const std::string& RUNPATH = "") const;
 
     ///\brief Concatenates current include paths and the system include paths
     /// and performs a lookup for the filename. If still not found it tries to
     /// add the platform-specific extensions (such as so, dll, dylib) and
     /// retries the lookup (from lookupLibInPaths)
     ///\param[in] filename - The filename being looked up
+    ///\param[in] RPATH - Library specific RPATH
```

Comment:
What does "library specific" mean? Is this maybe:
```suggestion
    ///\param[in] RPATH - RPATH as provided by another library, searching for libStem
```
? (Same comment for all other occurrences of "library specific")

Reply:
Should now be improved.

Reply:
Great thanks!